content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Tips for Submitting a Video Abstract to Posit Conf 2025"
+author: 
+  - Nic Crane
+  - Sayantika Banik
+date: '2025-02-03'
+---
+
+Thinking about submitting a talk for **Posit Conf 2025** but feeling nervous about recording a video abstract? You're not alone! It can feel intimidating at first, but the good news is—it's just **one minute** and all about showcasing your idea effectively.
+
+## **Tips from the R-Ladies Global Team**
+
+We know that recording a video can feel like a challenge, so we asked members of **R-Ladies Global** to share their best advice:
+
+📽️  **Shannon Pileggi**: Ask more than one person for feedback on your abstract and video content - everyone has a different and valuable point of view.Think about the way the principles of your content apply broadly to a general audience. [[Read Shannon's blog post talking about her experience of getting expected to speak at Posit Conf here](https://www.pipinghotdata.com/posts/2024-10-24-iterating-to-achieve-my-first-accepted-positconf-talk/)]
+
+💡 **Nic Crane**: Check out the [Posit blog post which tells you what they're looking for](https://posit.co/blog/how-we-build-the-positconf-program/) and has examples of past submissions.  Before recording, I try to get myself pumped up with energy by doing a dummy run pretending my idea is the most interesting thing I've heard in my entire life, with exaggerated enthusiasm, so then the normal version I do immediately afterwards has a bit of bounce to it.
+
+📽  *[Placeholder for a tip from another R-Ladies member]*
+
+🎤  *[Placeholder for a tip from another R-Ladies member]*
+
+💡  *[Placeholder for a tip from another R-Ladies member]*
+
+
```

Comment:
Some great tips here: https://www.linkedin.com/posts/rachaeldempsey_a-little-birdie-told-me-that-the-call-for-activity-7282863651554754563-Zy_I/

Reply:
Thanks so much!